### PR TITLE
homepage v_1

### PR DESCRIPTION
**Pull request by szmo**
https://trello.com/c/EwAeWCaB/17-homepage-first-approach

### CODE REVIEW

File: rails_app/app/assets/javascripts/application.js [hunk 1]
```diff
@@ -13,3 +13,5 @@
 //= require rails-ujs
 //= require turbolinks
 //= require_tree .
+//= require jquery
```
Comment (evemorgen) [inline rails_app/app/assets/javascripts/application.js:16]: Do we really need jQuery explicitly?
You might want to read this: https://css-tricks.com/now-ever-might-not-need-jquery/

File: rails_app/app/views/homepage/index.html.haml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.well
+  .row
+    .col-xs-6.col-xs-offset-3
+      .input-group
+        %input.form-control{type: 'text', placeholder: 'www.example.com'}
+        %span.input-group-btn
+          %button.btn.btn-default{type: 'button', type: 'button'}
+            Wyszukaj!
```
Comment (evemorgen) [inline rails_app/app/views/homepage/index.html.haml]: Dude, we're trying to keep this international, please stick to English

File: rails_app/app/views/layouts/application.html.haml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+    = stylesheet_link_tag 'application'
+    = javascript_include_tag 'application'
+    = csrf_meta_tags
+
+
+  %body
+    = render 'layouts/header'
+    - flash.each do |message_type, message|
+      .alert{ class: "alert-#{message_type}" }
```
Comment (evemorgen) [inline rails_app/app/views/layouts/application.html.haml:14]: Not sure what this alert does

File: rails_app/log/development.log [hunk 4]
```diff
@@ -1,7 +0,0 @@
-Started GET "/" for 127.0.0.1 at 2017-10-19 22:29:04 +0200
```
Comment (evemorgen) [inline rails_app/log/development.log:1]: ``` echo '**/*.log' >> .gitignore```

**Review by evemorgen — CHANGES_REQUESTED** [review]

File: rails_app/app/views/layouts/application.html.haml (hunk 3 above)
Comment (szmo) [reply rails_app/app/views/layouts/application.html.haml:14]: http://api.rubyonrails.org/classes/ActionDispatch/Flash.html

File: rails_app/app/assets/javascripts/application.js (hunk 1 above)
Comment (szmo) [reply rails_app/app/assets/javascripts/application.js:16]: Bootstrap JavaScript depends on jQuery. If you're using Rails 5.1+, add the jquery-rails gem to your Gemfile:

gem 'jquery-rails'

File: rails_app/app/views/homepage/index.html.haml (hunk 2 above)
Comment (szmo) [reply rails_app/app/views/homepage/index.html.haml]: Kk

File: rails_app/log/development.log (hunk 4 above)
Comment (szmo) [reply rails_app/log/development.log:1]: done

**Review by evemorgen — APPROVED** [review]

**Pull request merged**